src/pages/Thanks/ThanksPageIwdChallenge.vue

```diff
@@ -0,0 +1,282 @@
+					:icon="mdiCheckAll"
+				/>
+				<div v-if="receipt">
+					<p>
+						Success, your receipt has been sent to
+						<strong class="data-hj-suppress">{{ lender.email }}</strong>
+					</p>
+				</div>
+			</div>
+		</div>
+		<kv-page-container v-if="receipt">
+			<kv-grid class="tw-grid-cols-12 tw-pt-3 md:tw-pt-4 lg:tw-pt-6 tw-mb-4 md:tw-mb-6">
+				<div
+					class="tw-col-span-12 md:tw-col-span-6"
+				>
+					<img
+						src="@/assets/images/achievements/thankyou_badge@2x.jpg"
+						alt="Thank you badge"
+						class="tw-rounded tw-w-full tw--mb-1.5 md:tw--mb-1"
+					>
+				</div>
+				<!-- challenge complete copy -->
+				<div class="tw-col-span-12 md:tw-col-span-6">
+					<h2>Way to go!</h2>
+					<p class="tw-text-subhead tw-mt-1">
+						<!-- eslint-disable-next-line max-len -->
+						You earned the 2023 International Women’s Day badge by lending to a woman! Thank you for supporting gender equity worldwide.
+					</p>
+					<div
+						class="tw-bg-black tw-bg-opacity-[65%] tw-rounded tw-flex tw-items-center
+						tw-p-1 tw-mt-3 md:tw-mt-6"
+						style="max-width: 295px;"
+					>
+						<h3 class="tw-text-primary-inverse tw-mr-1.5">
+							You've completed your {{ achievementNumberLanguage }} challenge!
+						</h3>
+						<icon-sun class="tw-h-5 tw-w-5 tw-mr-0.5 tw-flex-none" />
+					</div>
+				</div>
+			</kv-grid>
+			<!-- Challenge Complete -->
+			<template>
+				<hr
+					class="tw-block tw-border-tertiary tw-w-full tw-mb-3"
+				>
+				<div
+					class="tw-flex tw-w-full tw-justify-between"
+				>
+					<h2 class="tw-mr-1.5">
+						Your {{ achievementNumberLanguage }} challenge!
+					</h2>
+					<div
+						class="tw-flex"
+					>
+						<icon-sun class="tw-h-6 tw-w-6 tw-mr-3 tw-flex-none tw-mt-0.5" />
+						<div>
+							<h3 class="tw-mb-1">
+								2023 Women's Day challenge
+							</h3>
+							<kv-progress-bar
+								aria-label="Completion of challenge"
+								:value="100"
+							/>
+							<p class="tw-text-h4 tw-text-right tw-font-medium tw-mt-1">
+								complete!
+							</p>
+						</div>
+					</div>
+				</div>
+				<div class="tw-mt-3 tw-mb-4 md:tw-mb-6">
+					<p class="tw-text-subhead" v-html="iwdFact">
+					</p>
+					<div class="tw-flex">
+						<kv-social-share-button
+							class="tw-mt-3"
+							modal-title="Spread the word about gender equity"
+							:share-message="iwdFact"
+							share-url="/gender-equality"
+							variant="primary"
+							utm-campaign="iwd_challenge_thanks_page"
+							style="max-width: 190px;"
+							tracking-category="post-checkout"
+						>
+							Share this fact
+							<template #modal-content>
+								{{ iwdFact }}
+							</template>
+						</kv-social-share-button>
+						<kv-button
+							class="tw-mt-3 tw-ml-3"
+							href="/gender-equality"
+							variant="secondary"
+							style="max-width: 190px;"
+						>
+							Learn More
+						</kv-button>
+					</div>
+				</div>
```

Comment:
It is possible to add a Portfolio link somewhere on this page like on our standard thanks page share version?

Reply:
Added in new version